### PR TITLE
DLP-4 Login page

### PR DESCRIPTION
**Pull request by Balazs14-hun**

### CODE REVIEW

File: resources/js/Components/RegisterLayout.vue [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+  <div class="my-28 content-center sm:my-0 sm:flex sm:h-screen sm:bg-gray-50">
     <RegisterSidePanel
       class="hidden xl:block"
       :registration-state="state"
     />
     <div class="m-auto flex-col justify-between space-y-10">
       <slot />
-      <div class="m-auto hidden w-fit space-x-4 sm:flex md:flex xl:hidden">
+      <div
+        v-if="state != 0"
```
Comment (PalZsolt99-ubb) [inline resources/js/Components/RegisterLayout.vue:12]: Just use **!state**

File: resources/js/Components/RegisterSidePanel.vue [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
   <BaseSidePanel>
-    <RegisterSidePanelItem
-      v-for="item in itemData"
-      :key="item.id"
-      :title="item.title"
-      :body="item.body"
-      :state="getState(item.id)"
-      :place="item.place"
-    />
+    <div v-if="registrationState != 0">
```
Comment (PalZsolt99-ubb) [inline resources/js/Components/RegisterSidePanel.vue:3]: Same here

**Review by PalZsolt99-ubb — CHANGES_REQUESTED** [review]

File: app/Http/Requests/LoginRequest.php [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        'email' => trans('auth.failed'),
+      ]);
+    }
+
+    RateLimiter::clear($this->throttleKey());
+  }
+
+  public function ensureNotRateLimited()
+  {
+    if (!RateLimiter::tooManyAttempts($this->throttleKey(), 7)) {
```
Comment (szabolcs3) [inline app/Http/Requests/LoginRequest.php:51]: Magic numbers are not allowed 😂 Put it into .env and read it

File: app/Http/Requests/LoginRequest.php [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  /**
+   * Get the validation rules that apply to the request.
+   *
+   * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+   */
+  public function rules(): array
+  {
+    return [
+      'email' => ['required', 'string', 'email'],
+      'password' => ['required', 'string'],
```
Comment (szabolcs3) [inline app/Http/Requests/LoginRequest.php:30]: We should put a `min:` and `max:` rule at least

File: app/Http/Requests/LoginRequest.php [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    if (!RateLimiter::tooManyAttempts($this->throttleKey(), 7)) {
+      return;
+    }
+
+    $seconds = RateLimiter::availableIn($this->throttleKey());
+
+    throw ValidationException::withMessages([
+      'email' => trans('auth.throttle', [
+        'seconds' => $seconds,
+        'minutes' => ceil($seconds / 60),
```
Comment (szabolcs3) [inline app/Http/Requests/LoginRequest.php:60]: Do we need this minutes? It's not used in `auth.php`

**Review by szabolcs3 — CHANGES_REQUESTED** [review]

**Pull request merged**